### DISCUSSION TITLE
Org level runners staying idle

### DISCUSSION THREAD

**Discussion by alexthecarp** [community/community#26473] [Actions]
<p>Hi all, </p>
<p>So we just deployed some Org level runners and they’re not picking up any tasks, even though the queues are full on one of the repos that’s already using repo-level runners.</p>
<p>I checked that the Org level runners are configured to be used by Private repos, which the one I’m testing with is. </p>
<p>Is there anything else I need to enable to make them pick up jobs ?</p>
<p>Thanks,</p>
<p>Alex</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/alexthecarp">@alexthecarp</a> ,</p>
<p>Please use a ‘unique’ label for orglevel self-hosted runner and specify it in your yaml file.</p>
<p>If there is only an org level self-hosted runner, workflow will pick it up.</p>
<p>However, if both repo level and org level runner exists, and use the same label in yaml file, workflow will use repo level runner, the org level runner will be idle.</p>
<p><img src="https://user-images.githubusercontent.com/52772761/181097941-934ee020-fa96-4e9c-84ed-a3b3d454c2b7.png" alt="orglevelrunnerlabel.png" data-base62-sha1="2sEOZ3sWhIBwJk3RCQCvfX5TBzD" width="677" height="364"></p>

**Comment by alexthecarp** [comment]
<p>Thank you! That did the trick.</p>
